src/adapter/src/coord/command_handler.rs

```diff
@@ -775,7 +779,7 @@ impl Coordinator {
         self.cancel_pending_peeks(conn.conn_id());
         self.end_session_for_statement_logging(conn.uuid());
         let update = self.catalog().state().pack_session_update(&conn, -1);
-        self.send_builtin_table_updates(vec![update]).await;
+        self.buffer_builtin_table_updates(vec![update]);
```

Comment:
I reverted this change and then re-added it because I noticed when tearing down many connections we could effectively hang the coordinator because it had to processes hundreds/thousands of table writes. I'm more than happy to go back to blocking on this change, but the effect it has on the coordinator did not seem great.

Reply:
I think it's fine, can we note it down in a comment here?

Reply:
Added a comment!